src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -36,6 +37,21 @@ interface LicenceRepository : JpaRepository<Licence, Long>, JpaSpecificationExec
     """,
   )
   fun getApprovedLicencesOnOrPassedReleaseDate(): List<Licence>
+
+  @Query(
+    """
+    SELECT l
+        FROM Licence l 
+        WHERE l.prisonCode IN :prisonCodes 
+        AND l.statusCode IN (
+            uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.ACTIVE,
+            uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.APPROVED
+        )
+        AND (COALESCE(l.actualReleaseDate, l.conditionalReleaseDate) > :releasedAfterDate)
+        ORDER BY l.conditionalReleaseDate ASC
+    """,
+  )
+  fun getRecentlyApprovedLicences(prisonCodes: List<String>?, releasedAfterDate: LocalDate): List<Licence>
```

Comment:
is there a reason why this is nullable? 

what happens if null is passed?

Reply:
I copied it from somewhere else, it doesn't make sense to be nullable here, fixed